cloud/edgecontroller/conf/controller.yaml

```diff
@@ -6,6 +6,7 @@ controller:
     qps: 5
     burst: 10
     node_update_frequency: 10
+    kubeconfig: ""   #Enter path to kubeconfig file to enable https connection to k8s apiserver
```

Comment:
I have left the default path empty as kubeclient gives first preference to kubeconfig file while creating the client. If user wants to use https connection, he can give path to kubeconfig file :)

Reply:
sounds fair.